src/imputors/drop.jl

```diff
@@ -29,25 +29,24 @@ end
 # TODO: Switch to using Base.@kwdef on 1.1
 DropObs(; context=Context()) = DropObs(context)
 
-function impute!(data::AbstractVector, imp::DropObs)
-    imp.context() do c
-        filter!(x -> !ismissing!(c, x), data)
-    end
+# Special case impute! for vectors because we know filter! will work
+function impute!(data::Vector, imp::DropObs)
```

Comment:
should we special case `::Matrix` too?

Reply:
No, because I don't think you can drop entire rows from a matrix in-place.

Reply:
ah, yeah, makes sense